pages/virtual-leagues/[type].tsx

```diff
@@ -35,11 +48,26 @@ const VirtualLeaguePage: NextPage<PageProps> = ({ res, verifiedType }) => {
 };
 
 VirtualLeaguePage.getInitialProps = async (ctx: NextPageContext) => {
-  const verifiedType = toVerifiedType(ctx.query.type as string);
-  var res = await getVirtualLeagues(verifiedType);
+  let verifiedType: VerifiedType = 'Unknown';
+
+  try {
+    verifiedType = toVerifiedType(ctx.query.type as string);
```

Comment:
denne kan ikke kaste exception, så du trenger ikke try/catch

Reply:
den gjør det etter at jeg endret den i denne PR 👯‍♂️ har du noen andre måter å try-parse på uten bruk av throw 'yolo' inne i `toVerifiedType()` ..?

Reply:
riiiight, der leste jeg litt fort! jeg tror kanskje jeg bare hadde handla `unknown` som ikke valid verified-type, og fortsatt hatt det som default da, for exceptions gir ikke noe warning om at de må håndteres. Evt. hvis du vil være veldig fancy, så kunne du return type som var `VerifiedType | IllegalVerifiedType`, eller bare `VerifiedType | null`

Reply:
Ok, var kanskje ikke helt klart hva jeg prøvde få til, men: 

- Ønsker at "Unknown" behandles likt som andre types, sånn at vi kan (skjult, ikke listet ut i index) se de på https://test.fplbot.app/virtual-leagues/unknown
- Ønsker at alle andre ting i url som types inn gir en 404. typ https://test.fplbot.app/virtual-leagues/hansens-super-liga

Reply:
Ye, da må du gjøre "fancy"-metoden. Grunnen til at jeg ikke liker exception er fordi da må andre som bruker funksjonen bare *huske* at de må håndtere det, mens hvis du bruker return-typen til å vise alle cases som må håndteres får man ikke lov å TS å ikke håndtere alle

Reply:
Right, så bruke `null` istedet for exceptions og try-catch. Enig at det er penere. Fikser.

Reply:
Fiksa i 6b5eb2c